json_rpc/router.nim

```diff
@@ -229,6 +227,15 @@ proc tryRoute*(router: RpcRouter, data: JsonString,
     {.pop warning[BareExcept]:on.}
     {.pop warning[UnreachableCode]:on.}
 
+proc tryRoute*(router: RpcRouter, data: JsonString,
+               fut: var Future[JsonString]): Result[void, string] =
+  ## Route to RPC, returns false if the method or params cannot be found.
+  ## Expects json input and returns json output.
+  try:
+    let req = JrpcSys.decode(data.string, RequestRx)
```

Comment:
Why the original `tryRoute` not called here? That is exactly what caused the failures.

Reply:
Ah right, a diff issue. Sorry about that